src/Stratis.Bitcoin.Networks/Deployments/BIP9Deployments.cs

```diff
@@ -0,0 +1,50 @@
+﻿using NBitcoin;
+
+namespace Stratis.Bitcoin.Networks.Deployments
+{
+    /// <summary>
+    /// BIP9 deployments for the Bitcoin network.
+    /// </summary>
+    public class BIP9Deployments : BIP9DeploymentsArray
+    {
+        // The position of each deployment in the deployments array.
+        public const int TestDummy = 0;
+        public const int CSV = 1;
+        public const int Segwit = 2;
+
+        // The number of deployments.
+        public const int NumberOfDeployments = 3;
+
+        /// <summary>
+        /// Constructs the BIP9 deployments array.
+        /// </summary>
+        public BIP9Deployments() : base(NumberOfDeployments)
+        {
+        }
+
+        /// <summary>
+        /// Gets the deployment flags to set when the deployment activates.
+        /// </summary>
+        /// <param name="deployment">The deployment number.</param>
+        /// <param name="scriptFlags">The script flags to set.</param>
+        /// <param name="lockTimeFlags">The lock time flags to set.</param>
+        public override void GetFlags(int deployment, out ScriptVerify scriptFlags, out Transaction.LockTimeFlags lockTimeFlags)
```

Comment:
What about returning a new class here with the two params in the class, instead of out params? WDYT

Reply:
Never mind ignore this comments lets try to keep changes to a minimum

Reply:
Done before I saw your second comment. I hope that's ok.